src/buffer/out/CharRowCell.hpp

```diff
@@ -27,8 +28,7 @@ Author(s):
 class CharRowCell final
 {
 public:
-    CharRowCell() noexcept;
-    CharRowCell(const wchar_t wch, const DbcsAttribute attr) noexcept;

```

Comment:
This one was unused?

Reply:
I had been trying to make this type into a POD since that was looking even better for perf, but it was too hard to get it correct for tests so I gave up.  I could restore this if you'd like, since the POD experiment didn't work out.

Reply:
I don't mind cleanup, but with a 400+ file PR whose focus is supposed to be allocations... I'm a mild bit grumpy at sneaking in other cleanup, even if it's nice to have.

Reply:
Yeah, fair enough.  I'll double-check that I didn't get any packing wins and if not, I can back this out to help the PR be more manageable-sized.

Reply:
The unsigned int changes didn't save anywhere except in TextAttributeRun - so I kept it there, but tried to reduce the spread by using gsl::narrow inside the implementation of TextAttributeRun.  Unless any specific run is above 4 billion characters, this will never throw, so it seems pretty safe and reduces churn in the PR.

Reply:
OK with the reduced size to this PR thanks to separating boost... I'm fine with leaving this cleanup in even if it didn't completely POD out.